PIL/features.py

```diff
@@ -5,6 +5,7 @@
     "tkinter": "PIL._imagingtk",
     "freetype2": "PIL._imagingft",
     "littlecms2": "PIL._imagingcms",
+    "raqm": "PIL._imagingft",
```

Comment:
This is going to trigger on raqm being available when it's not compiled in but freetype is.